main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1054,15 +1136,17 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
   }
 
   /**
-    * Builds a nested map of namespace -> name -> signature from the given class namespace map..
+    * Builds a nested map of namespace -> name -> signature from the given class namespace map.
     */
-  def buildSigLookup(classes: Map[Name.NName, Map[String, NamedAst.Class]]): SigLookup = {
+  def buildSigLookup(classes: Map[Name.NName, Map[String, NamedAst.Class]]): Map[Name.NName, Map[String, NamedAst.Sig]] = {
```

Comment:
I would think that something like this could go in NamedAst and be constructed in the Namer. Is that possible?